fdb-record-layer-core/src/test/java/com/apple/foundationdb/record/query/plan/cascades/values/simplification/ValueComputationTest.java

```diff
@@ -174,7 +174,7 @@ void testPullUpValue2() {
         final var new_x_b = FieldValue.ofFieldNames(upperCurrentValue, ImmutableList.of("_1", "x", "xb"));
 
         final var expectedMap = ImmutableMap.of(
-                record_type, new RecordTypeValue(QuantifiedObjectValue.of(UPPER_ALIAS, new Type.AnyRecord(true))),
+                record_type, new RecordTypeValue(FieldValue.ofFieldName(upperCurrentValue, "_1")),
```

Comment:
What was the logic in changing this?

Reply:
The `RecordTypeValue(...)` has a child but never advertised it causing other trouble. Correcting this lead me to realize that this test case expected the wrong output all along.